openc3/lib/openc3/models/cvt_model.rb

```diff
@@ -82,7 +84,12 @@ def self.get_item(target_name, packet_name, item_name, type:, scope:)
       end
       hash = JSON.parse(Store.hget("#{scope}__tlm__#{target_name}", packet_name), :allow_nan => true, :create_additions => true)
       hash.values_at(*types).each do |result|
-        return result if result
+        if result
+          if type == :FORMATTED or type == :WITH_UNITS
+            return result.to_s
+          end
+          return result
+        end
```

Comment:
We were breaking the contract of FORMATTED and WITH_UNITS that they are always a String